pkg/operator/ceph/object/user/controller.go

```diff
@@ -493,7 +491,7 @@ func updateStatus(client client.Client, name types.NamespacedName, status string
 		return
 	}
 	if user.Status == nil {
-		user.Status = &cephv1.Status{}
+		user.Status = &cephv1.ObjectStoreUserStatus{}
```

Comment:
Maybe I'm missing it, but I don't see the secret name being updated in the `ObjectStoreUserStatus`.

Reply:
> Maybe I'm missing it, but I don't see the secret name being updated in the `ObjectStoreUserStatus`.

Done.